app/controllers/npq_separation/admin/bulk_operations/base_controller.rb

```diff
@@ -40,12 +40,8 @@ def find_bulk_operation
       @bulk_operation = bulk_operation_class.find(params[:id])
     end
 
-    def sort_order
-      %i[created_at id]
-    end
-
     def set_bulk_operations
-      @bulk_operations = bulk_operation_class.all.includes([file_attachment: :blob]).order(sort_order)
+      @pagy, @bulk_operations = pagy(bulk_operation_class.all.includes([file_attachment: :blob]).order(created_at: :desc, id: :desc))
```

Comment:
Can we simplify this line to reduce reading complexity - either assign to a variable or move the data load part to a separate proviate method this calls